app/views/expenses/_form.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= simple_form_for expense do |f| %>
+  <%= f.input :name %>
+  <%= f.input :due_date, order: [:day, :month, :year] %>
+  <%= f.input :frequency %>
+  <%= f.input :bill_estimate, input_html: { value: number_with_precision(f.object.bill_estimate, precision: 2) } %>
```

Comment:
Do you think this is clear enough for the use? or would they benefit from using a placeholder value here?

Reply:
A placeholder wouldn't hurt! Done in sha: 81b92cf